### DISCUSSION TITLE
Custom CSS in BetterSEQTA+

### DISCUSSION THREAD

**Discussion by Crazypersonalph** [SethBurkart123/EvenBetterSEQTA#62] [Ideas]
As of right now, we have the ability to have custom backgrounds and custom themes, but we do not have the ability to actually configure the CSS of the project ourselves.

I was already working on implementing custom CSS on HyperSEQTA before I merged it, so now I guess I will continue my efforts here.

**Comment by SethBurkart123** [comment]
Honestly, this could be a cool feature. I was thinking it would be cool if we made it possible for people to create and share their themes... It might be hard figuring out servers - although if we used something like the free tier of supabase? Maybe we could do a store (well more like a free store, but you know) that allows people to share them?

(I could throw together the store quite fast as I have experience with stuff like that)

**Reply by MEGA-Dawg68** [reply]
Why don't we host on GitHub pages? it is free with no paid tier (At least I don't think so)

**Reply by SethBurkart123** [reply]
Yeah, it is free and great, but the only thing is how people post themes. Whether they'd have to go onto github and do a technical submission. It's possible but not preferable. I was thinking kind of a combination of the both. You use github pages (or the chrome extension) to host the frontend and then there is a backend. For the only purpose of having a way for people to post their own themes...

Not sure, but it's an idea...

**Reply by MEGA-Dawg68** [reply]
I use Github pages to host my website, I don't really have much experience so I didn't know

**Reply by SethBurkart123** [reply]
It's not too hard. We could do it if need be.

**Comment by SethBurkart123** [comment]
Well, we'd need a code editor inside the web so that they can edit live...

https://github.com/microsoft/monaco-editor

Why not use the editor that powers vs-code? Keeps it nice, simple for us, but very clean and powerful for the user.

**Reply by MEGA-Dawg68** [reply]
Is it lightweight? VS code lags like hell on my school laptop lol

**Reply by SethBurkart123** [reply]
yeah, it will be lightweight enough, because you only will have the code editor when you are writing the actual css. And it's only one instance, it's not like it's heaps of files open at once with plugins loaded etc.

**Reply by MEGA-Dawg68** [reply]
> it's not like it's heaps of files open at once with plugins loaded etc.

That might be my problem lol
